### DISCUSSION TITLE
Can someone explain LiteLLM format?

### DISCUSSION THREAD

**Discussion by GanizaniSitara** [open-webui/open-webui#949] [General]
Hi, I'm sorry, I'm having a torrid time getting this running on a firewalled network. I'm on the same revision of otherwise unaltered codebase, in one I'm seeing the model line as "Ener LiteLLM Model" (line 706 in Modesl.svelte), on the other I'm getting "Enter model tag (e.g. mistral:7b)"  (line 378 in the same file). 

What decides which format is used? The former is coming up on the firewalled network and doesn't work. I previously had a way to bring in models by hand so if this uses different method I'm snookered. Pointers will be much appreciated. Thanks. GS

**Comment by justinh-rahb** [comment]
I think in your former case your WebUI is not connected to an Ollama server, so it is not showing the Ollama models part of the **Models** tab. Probably your firewall is preventing the WebUI from communicating with your Ollama API.

**Reply by GanizaniSitara** [reply]
Ollama is local and works, no firewall there, ostensibly same config on both machines ... well you might be onto something one is in WSL the other windows, hmm will double-check

**Reply by justinh-rahb** [reply]
Between WSL and the host, the idea of **local networks** becomes... *strained*. Most likely you can resolve this by assuming that you are not on the same Layer2 network segment and working accordingly. The Ollama server by default only listens on 127.0.0.1, so you need to set environment variables `OLLAMA_HOST=0.0.0.0` and possibly also `OLLAMA_ORIGINS=*`, [per their FAQ](https://github.com/ollama/ollama/blob/main/docs/faq.md#setting-environment-variables-on-windows).
